### ISSUE TITLE
Bug when loading timeseries 

### ISSUE THREAD

**Issue by jnnr** [rl-institut/oemof-B3#92]
When I try to load the data prepared by the rule `prepare_feedin` (or `prepare_electricity_demand` equally), like this:

```
from oemof_b3.tools.data_processing import load_b3_timeseries

df = load_b3_timeseries("results/_resources/feedin_time_series.csv")
```
I get this error
```
ValueError: There are extra columns ['Unnamed: 0']
```

Which is due to the fact that the index column is not named correctly. When I fix this (by naming the column manually, for the moment), I get another error:

```
File "oemof-B3/oemof_b3/tools/data_processing.py", line 133, in <lambda>
  df.loc[:, "series"] = df.loc[:, "series"].apply(lambda x: ast.literal_eval(x), 1)

SyntaxError: unexpected EOF while parsing
```

Do you understand how this happens and how to fix it, @MaGering?

**Comment by MaGering** [comment]
I cannot reproduce this error, as `prepare_feedin` and `prepare_electricity_demand` runs through without error for me. I guess the index of your time series shouldn't be named, because it gets the name `id_ts` in `format_header` function. If on the other side the index has a name that does not match `id_ts`, `format_header` interprets the `format_header` as an unknown column and prints this error. Does this help you?

**Comment by jnnr** [comment]
The two rules pass for me, too. However, I am not able to load the resulting data with the code snipped provided above.

**Comment by MaGering** [comment]
This can be fixed with setting `index=False` in [`save_df` function](https://github.com/rl-institut/oemof-B3/blob/e0727d7b37ef16043fdd135b7213120fa4e239b9/oemof_b3/tools/data_processing.py#L147).

With this adaptation I could read the result data and the tests still passed. So it doesn't seem to affect the handling of other files at first sight.

Should I do a PR for this?

**Comment by jnnr** [comment]
> This can be fixed with setting `index=False` in [`save_df` function](https://github.com/rl-institut/oemof-B3/blob/e0727d7b37ef16043fdd135b7213120fa4e239b9/oemof_b3/tools/data_processing.py#L147).
> 
> With this adaptation I could read the result data and the tests still passed. So it doesn't seem to affect the handling of other files at first sight.
> 
> Should I do a PR for this?

This is an approach to the first error described above. To decide what to do here, we should first decide: Should the load and format function be tolerant:

* if data has no numeric index?
* If there is a numeric index as a first column, but it has no name like "id_scal" or "id_ts"?

Or should it fail and give an informative error message?

**Comment by jnnr** [comment]
Interesting that you cannot reproduce the second error. I still see it and it has to be solved.

**Comment by jnnr** [comment]
It seems that the saved timeseries can be opened, however, when the entries in `series` are interpreted by ast.literal_eval, the error occurs. This is because the entries of `series` are missing a final closing bracket. They look like this `"[1,1,"`. Have to check why this happens.

**Comment by jnnr** [comment]
The first error (unnamed index is resolved with 100282f). The other error could not be reproduced, therefore I close this issue. Reopen if it reoccurs.
